documentation/api/metrics/v1/mbeans.markdown

```diff
@@ -152,4 +152,43 @@ resources, storing edges, etc.). Metrics of particular note include:
 
 [Using `curl` from localhost][curl]:
 
+    curl 'http://localhost:8080/metrics/v1/mbeans/puppetlabs.puppetdb.command%3Atype%3Dglobal%2Cname%3Dprocessing-time'
```

Comment:
let this one be because it's inconsistent with the rest of our docs -- if we want to make a broader change with the prompt we can talk about it, but I'd be against because it requires more aim to copy-paste